### PR TITLE
Update/tree pin

### PR DESCRIPTION
**Pull request by sjaymoon15**
- Added .env for the client app. 
- set REACT_APP_WEBMAP_DOMAIN variable as `https://treetracker.org`. 
- Added TreePin svg file.
- Added handleTreePinClick function for handling clicking the TreePin image events.

Related Issue: #66

### CODE REVIEW

**Comment by sjaymoon15** [comment]
I am noticing I might have different prettier setting than others. (or some might not have prettier set up). If I am not aligned with the team's setting, please let me know so I can adjust.

**Comment by dadiorchen** [comment]
hi, @sjaymoon15 thanks for your work, because seems like you re-formatted the source code, the comparison shows too many differences, it's a bit of hard to tell what did you changed, let me ask you directly: the main work of this PR is: you added a button and when click on it, will open a new window to show the tree detail, right?

**Comment by sjaymoon15** [comment]
Yes @dadiorchen .Pretty much. 
I added a button (Tree Pin image 'button') and a function to handle that button click event called `handleTreePinClick`
That function opens a new browser tab and directs users to the tree in Webmap app. 

Will add comments where I made changes.

File: client/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_WEBMAP_DOMAIN=https://treetracker.org
```
Comment (sjaymoon15) [inline client/.env:1]: created a .env file and added REACT_APP_WEBMAP_DOMAIN variable.

File: client/src/components/TreeImageScrubber.js [hunk 2]
```diff
@@ -59,224 +61,250 @@ const useStyles = makeStyles(theme => ({
...
+    e.stopPropagation();
+    e.preventDefault();
+    log.debug('click at tree:%d', treeId);
+    props.verityDispatch.clickTree({
+      treeId,
+      isShift: e.shiftKey
+    });
+  }
+
+  function handleTreePinClick(e, treeId) {
```
Comment (sjaymoon15) [inline client/src/components/TreeImageScrubber.js:152]: Event handler for Clicking tree pin image.

File: client/src/components/TreeImageScrubber.js [hunk 3]
```diff
@@ -59,224 +61,250 @@ const useStyles = makeStyles(theme => ({
...
+                      }
+                    });
+                  }}
+                  disabled={tree.approved === true}
+                >
+                  Approve
+                </Button>
+              </Box>
+              <Box>
+                <TreePin
```
Comment (sjaymoon15) [inline client/src/components/TreeImageScrubber.js:243]: Added TreePin image/button

File: client/src/components/TreeImageScrubber.js [hunk 4]
```diff
@@ -59,224 +61,250 @@ const useStyles = makeStyles(theme => ({
...
+  title: {
+    padding: theme.spacing(2, 16)
+  },
+  snackbar: {
+    bottom: 20
+  },
+  snackbarContent: {
+    backgroundColor: theme.palette.action.active
+  },
+  cardActions: {
```
Comment (sjaymoon15) [inline client/src/components/TreeImageScrubber.js:75]: added css classes
`cardActions` and `button` classes.

**Comment by dadiorchen** [comment]
@sjaymoon15 thanks!

**Pull request merged**